### PR TITLE
CMCL-1540: Add doc generation job for Cinemachine

### PR DESCRIPTION
**Pull request by windxu88**
[Delete any line or section that does not apply]

### Purpose of this PR
CI: Add a doc generation job for Cinemachine

Sebastien needs a CI job which can generate documentation for Cinemachine's main branch so that he can check whether the doc he put is right or not. Without this CI job, he needs to generate documentation manually which is time consuming and boring.
Please have a look at [the artifacts of Generate documentation job](https://unity-ci.cds.internal.unity3d.com/job/32369509/artifacts), `documentation/Package_Doc->index.html`.

[JIRA issue. Desc of feature/change. Links to screenshots, design docs, user docs, etc. Remember reviewers may be outside your team, and not know your feature/area that should be explained more.]
[CMCL-1540](https://jira.unity3d.com/browse/CMCL-1540) CI: Add a documentation generation job for Cinemachine

### Testing status

[Explanation of what’s tested, how tested and existing or new automation tests. Can include manual testing by self and/or QA. Specify test plans. Rarely acceptable to have no testing.]

- [X] Added an automated test
- [ ] Passed all automated tests
- [ ] Manually tested 

### Documentation status

[Overview of how documentation is affected by this change. If there is no effect on documentation, explain why. Otherwise, state which sections are changed and why.]

- [ ] Updated [CHANGELOG](https://keepachangelog.com/en/1.0.0/)
- [ ] Updated README (if applicable)
- [ ] Commented all public classes, properties, and methods
- [ ] Updated user documentation

### Technical risk

[Overall product level assessment of risk of change. Need technical risk & halo effect.]

### Comments to reviewers

[Info per person for what to focus on, or historical info to understand who have previously reviewed and coverage. Help them get context.]

### Package version

[Justification for updating either the patch, minor, or major version according to the [semantic versioning](https://semver.org/spec/v2.0.0.html) rules]

- [ ] Updated package version

### CODE REVIEW

**Comment by glabute** [comment]
What is this PR doing?

**Comment by glabute** [comment]
Makes sense, thanks.  Maybe put that text in the PR description?

**Review by sebastienduverne — APPROVED** [review]
Great, thanks!

**Comment by windxu88** [comment]
@AntoineCharton @glabute Could I get another approval so that I can merge this PR? Thanks

**Review by glabute — APPROVED** [review]

File: .yamato/package-documentation.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{% metadata_file .yamato/metadata.metafile %}
+---
+
+generate_documentation:
+  name : Generate documentation
+  agent:
+    type: Unity::VM::osx
+    image: package-ci/macos-13:v4
```
Comment (AntoineCharton) [inline .yamato/package-documentation.yml:8]: Do we sepcifically need a mac?

**Review by AntoineCharton — APPROVED** [review]
Looks good. Not sure why we need a mac VM. If not maybe use a linux but other than that good to go.

File: .yamato/package-documentation.yml (hunk 1 above)
Comment (windxu88) [reply .yamato/package-documentation.yml:8]: Ubuntu image is also OK to run the brick, but based on my experience, Mac VM without GPU is more available than other typed of VMs, so it has a shorter waiting time.

**Comment by windxu88** [comment]
Oh, it turns out that I am not an authorized user to merge the PR.
@glabute @AntoineCharton Could any of you help merging my PR? Thanks!

**Pull request merged**